### PR TITLE
CIP-0077? | Verified Stake Pool Identity

### PR DESCRIPTION
**Pull request by Maetti79**
The current CIP ID is set to 0071, but this can be changed.
(in accordance with CIP0066)

### CODE REVIEW

**Comment by rphair** [comment]
thanks for submitting @Maetti79 - looks good after a light read (more than enough to be sure this is distinct from https://github.com/cardano-foundation/CIPs/pull/241).

> The current CIP ID is set to 0071, but this can be changed.

Generally I would recommend all numbers can be removed until we have at least a chance to confirm that this is being entered as a Candidate CIP.  But it will be immediately helpful if the number in the current subdirectory `CIP-0071` and the number currently in the header `57` are made consistent... preferably by removing both numbers (with a `?` in place of a number in the header) 🤓

**Comment by Maetti79** [comment]
Thanks for changing and the clarification, that helped me a lot.

File: CIP-0071/README.MD [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP defines a standard to verify Staking Pools using Decentralized Identifiers (DID)
+
+## Motivation
+
+The verification process of the authenticity of Staking Pools is currently a centralized effort.
```
Comment (KtorZ) [inline CIP-0071/README.MD:17]: May you give perhaps a use case / concrete example of what "the verification process" looks like at the moment and who are the central authority doing this? (I am genuinely not aware of any?)

File: CIP-0071/README.MD [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+## Abstract
+
+This CIP defines a standard to verify Staking Pools using Decentralized Identifiers (DID)
+
+## Motivation
+
+The verification process of the authenticity of Staking Pools is currently a centralized effort. 
+
+## High-Level Overview
```
Comment (KtorZ) [inline CIP-0071/README.MD:19]:
```suggestion
## Specification

### High-Level Overview
```

File: CIP-0071/README.MD [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+         "storageLocation":{
+            "uri":"ipfs://QmaCnFpHZqC6vUfVobUscCxRFn1snb3XH7K8W3ZEAkooeo",
+            "type":"ipfs"
+         }
+      }
+   ]
+}
+```
+
+#### 3. Extended Stake-Pool metadata
```
Comment (KtorZ) [inline CIP-0071/README.MD:178]: There's no 2.

File: CIP-0071/README.MD (hunk 1 above)
Comment (SebastienGllmt) [reply CIP-0071/README.MD:17]: Currently CIP6 is kind of a verification scheme: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0006

File: CIP-0071/README.MD (hunk 3 above)
Comment (SebastienGllmt) [inline CIP-0071/README.MD:178]: It's probably good to specify this is specifically referring to CIP6

**Comment by SebastienGllmt** [comment]
I would say there are to ways to do this:
1. Have the metadata specify the identity
2. Have the identity specify the metadata

I think this CIP is aiming for (1), but maybe it's best to go with (2)

File: CIP-0071/README.MD [hunk 4]
```diff
@@ -0,0 +1,238 @@
+---
+CIP: 0071
```
Comment (KtorZ) [inline CIP-0071/README.MD:2]:
```suggestion
CIP: 71
```

File: CIP-0071/README.MD [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+Title: Verified Pool Identity
+Authors: Dennis Mittmann <dennis@iamx.id>
+Status: Active
```
Comment (KtorZ) [inline CIP-0071/README.MD:5]:
```suggestion
Status: Proposed
```

File: CIP-0071/README.MD [hunk 6]
```diff
@@ -0,0 +1,238 @@
+---
+CIP: 0071
+Title: Verified Pool Identity
```
Comment (KtorZ) [inline CIP-0071/README.MD:3]:
```suggestion
Title: Verified Stake-Pool Identity
```

**Review by rphair — COMMENTED** [review]
So far by authors & reviewers we have different permutations of the canonical term **stake pool** as "staking pools", "stake-pool(s)", etc.  It should be spelled everywhere with the canonical term, unless anyone knows it's spelled otherwise in a more definitive source than this document: [Cardano Docs > Stake pools](https://docs.cardano.org/learn/stake-pools)

File: CIP-0071/README.MD (hunk 6 above)
Comment (rphair) [reply CIP-0071/README.MD:3]: don't want to nitpick after my last review comment, only to address comments so far: there is no hyphen in "stake pool" (reference link below)

**Comment by rphair** [comment]
Great job @Maetti79 on recent round of changes & the candidacy of this proposal in general.

Editorially I had to eliminate the misspelled term "stake-pool" which has no hyphen: I didn't hold this up for permission or review because in assembling these documents we can't indugle spelling mistakes which will lead to other spelling mistakes by imitation.  Likewise about the colloquial but incorrect term "staking pool".  (Technically this term shouldn't be capitalised either, but punctuation mistakes are less likely to be imitated than spelling mistakes.)

**Comment by Maetti79** [comment]
Further and recent information about how and why is available here:

https://medium.com/iamx-own-your-identity/blockchain-compliance-proof-of-stake-regulation-9a4b6490448a

**Comment by rphair** [comment]
@Maetti79 now that official number `0077` is assigned, please also rename the proposal folder in your branch from `CIP-0071` to `CIP-0077` 😎

File: CIP-0071/README.MD [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+## Backwards Compatibility
+
+To keep metadata compatible with changes coming up in the future, we use the **`version`** property.
+
+## Rationale
+The focus of this standard is on ease-of-use & flexibility. Using this standard, it is possible to reference DIDs not only on Cardano but at any arbitrary location, even cross-chain. It is also relatively simple to understand and implement because it borrows heavily from pre-existing metadata standards.
+
+## Path to Active
+
+By displaying this solution at the Catalyst Guild Hall, we hope to reach more creators that are willing to integrate this solution.
```
[the remark this answers is not in the document]
Comment (rphair) [reply CIP-0071/README.MD:229]: As a current SPO I would rather state more generally that this is a "proposal which _appeal to a subset of_ stake pool operators."

**Review by rphair — COMMENTED** [review]
@Maetti79 also: after the last round of commits, is this proposal up for editorial & peer review again?  There are some organisation & formatting issues apparent, but if you had further work planned and/or were going to collaborate with authors & advocates for CIP-0006 as @KtorZ [suggested](https://github.com/cardano-foundation/CIPs/pull/361#discussion_r1016985655) then I would happily wait before listing them in detail.

Some tentative feedback, without a full code review:
- The points in "High-Level Overview" alternately apply to Motivation, Specification and Rationale and should be moved into those sections.
  - Some if not all of those abbreviated / bulleted points should be filled out into explanatory text.
- Since references to regulatory sources seem to be in the works (from your [recent comment above](https://github.com/cardano-foundation/CIPs/pull/361#issuecomment-1433113295)) it will help to either spell the terms Know-Your-Customer and Know-Your-Business correctly (no hyphens) or eliminate them from the unnecessary Glossary.
- Unofficial sections Cross-Chain Compatibility, Backwards Compatibility, and References should also be subsumed into official sections (probably all parts of Specification).
- Path to Active needs to have 2 sections for Acceptance and Implementation [as described in latest version of CIP-0001](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#path-to-active)... I think @KtorZ 's [suggestion](https://github.com/cardano-foundation/CIPs/pull/361#discussion_r1016985655) is a good one: to have SPO interest and adoption appearing as items on the checklist(s) here.

The technical part of the Specification seems (to me, not a metadata expert) well done: we just need to make sure it's bracketed with writing that puts this into context as a potential standard.

**Comment by rphair** [comment]
@Maetti79 this proposal is on the agenda for the CIP Editors' Meeting today... if you can be there please do (and post if you need directions; it's in less than 90 minutes).  I'm marking "Waiting for author" because there's been no response yet to the feedback above & no comments from the SPOs who would be the mainly interested parties for this.

Personally I think it that having this CIP on the books would be an asset in the "Contingent Staking" dialogue that flared up in the last month.  If you can continue the work on this proposal I feel sure the community would welcome it.  (cc @KtorZ @Ryun1)

**Comment by Maetti79** [comment]
I am available, where should i go/be, is there a meeting?

**Comment by rphair** [comment]
@Maetti79 CIP Editors Meeting Discord invite: https://discord.gg/kyaTyzkBqd
Meeting link: https://discord.com/events/971785110770831360/1078617056754286632

**Comment by rphair** [comment]
@Maetti79 thanks for coming to the CIP meeting today & please, as we said in the meeting, respond to the feedback above and we'll review it on GitHub in the meantime and progress it more at the meeting.  Attendees agreed that this would be a timely proposal given the current issues of providing identity & uniqueness.

@gufmar we also were hoping (suggested by @KtorZ at meeting today) you can interface with the SPO community and get some operator discussion on this?  I asked for this on GitHub itself above but the reach was pretty small & we only got 1 response... can get this in the regular SPO call agenda?

**Comment by rphair** [comment]
@Maetti79 I'm changing this to an `Abandoned` status since it's been over a year without a response to the last feedback... it'll be closed soon afterwards as such if there aren't any further updates within a reasonable time frame to respond.

**Pull request closed without merging**